python/recipe.hpp

```diff
@@ -74,18 +74,6 @@ class py_recipe_trampoline: public py_recipe {
         PYBIND11_OVERLOAD_PURE(arb::cell_kind, py_recipe, cell_kind, gid);
     }
 
-    arb::cell_size_type num_sources(arb::cell_gid_type gid) const override {
-        PYBIND11_OVERLOAD(arb::cell_size_type, py_recipe, num_sources, gid);
-    }
-
-    arb::cell_size_type num_targets(arb::cell_gid_type gid) const override {
-        PYBIND11_OVERLOAD(arb::cell_size_type, py_recipe, num_targets, gid);
-    }
-
-    arb::cell_size_type num_gap_junction_sites(arb::cell_gid_type gid) const override {
-        PYBIND11_OVERLOAD(arb::cell_size_type, py_recipe, num_gap_junction_sites, gid);
-    }
-
```

Comment:
`num_sources`, `num_targets`, and `num_gap_junction_sites` should be removed from `py_recipe` too.
A number of recipes in the python examples still have `num_xxx` methods defined.